src/Plugin/InstallTask/SiteSettings.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Drupal\stanford_profile\Plugin\InstallTask;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Site\Settings;
+use Drupal\externalauth\AuthmapInterface;
+use Drupal\stanford_profile\InstallTaskBase;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\GuzzleException;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * SNOW site settings installation.
+ *
+ * @InstallTask(
+ *   id="stanford_profile_site_settings"
+ * )
+ */
+class SiteSettings extends InstallTaskBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The fallback site name.
+   */
+  const DEFAULT_SITE = 'default';
+
+  /**
+   * Service now api endpoint.
+   */
+  const SNOW_API = 'https://stanford.service-now.com/api/stu/su_acsf_site_requester_information/requestor';
```

Comment:
There are different credentials and endpoints for their testing service.

Reply:
i just followed this https://github.com/SU-SWS/Stanford-Drupal-Profile/blob/7.x-3.x/stanford.profile#L447

Reply:
so the variable could change is what you're saying?

Reply:
now it can change. yay state https://github.com/SU-SWS/stanford_profile/pull/30/files#diff-6433294d9f56756da9c573358d444bd1R162

Reply:
https://github.com/SU-SWS/acsf-cardinald7/blob/1.x/factory-hooks/post-settings-php/snow-api-creds.php

Reply:
@pookmish This needs to be configurable as the endpoints differ vs dev/test & prod.

Reply:
Nevermind, I see the setting get.